move-vm-backend/src/lib.rs

```diff
@@ -36,6 +37,36 @@ where
     warehouse: Warehouse<S>,
 }
 
+/// Call type used to determine if we are calling script or function inside some module.
+#[derive(Debug)]
+pub enum Call {
```

Comment:
I think we can strip the `pub` keyword now from all these struct/enums below
```suggestion
enum Call {
```

Reply:
Done.